src/MTGCore.ClientApp/ClientApp/src/App.vue

```diff
@@ -44,138 +45,156 @@
               <div class="row">
                 <div class="col-lg-6">Cards in Deck</div>
                 <div class="col-lg-6">
-                  <DeckList ref="deckListRef"
-                            v-on:triggerChange="updateDeckCardList"
-                            v-on:addDeck="addNewDeck"
-                            v-bind:deckList="deckList"
-                            v-bind:selectedOption="selectedDeck"/>
+                  <DeckList
+                    ref="deckListRef"
+                    v-on:triggerChange="updateDeckCardList"
+                    v-on:addDeck="addNewDeck"
+                    v-bind:deckList="deckList"
+                    v-bind:selectedOption="selectedDeck"
+                  />
                 </div>
               </div>
             </div>
             <div class="card-body">
-              <DeckCardList v-bind:deckCards="deckCards"/>
+              <DeckCardList v-bind:deckCards="deckCards" />
             </div>
           </div>
         </div>
+        <SearchFilterModal v-on:submitFilter="SetSearchFilter" ref="SearchFilterRef"></SearchFilterModal>
       </div>
     </div>
   </div>
 </template>
 
 <script>
-import ResultsTable from "./components/ResultsTable.vue";
-import DeckCardList from "./components/DeckCardList.vue";
-import DeckList from "./components/DeckList.vue";
+import ResultsTable from './components/ResultsTable.vue';
+import DeckCardList from './components/DeckCardList.vue';
+import DeckList from './components/DeckList.vue';
+import SearchFilterModal from './components/SearchFilterModal.vue';
 import './assets/css/Main.css';
 
-import "bootstrap";
-import "bootstrap/dist/css/bootstrap.min.css";
-import axios from "axios";
+import 'bootstrap';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import axios from 'axios';
 
 export default {
-  name: "app",
+  name: 'app',
   data: function () {
     return {
-      Name: "",
+      Name: '',
       searchedCards: null,
-      searchedCardsLoadingError: "",
+      searchedCardsLoadingError: '',
       searchInProgress: false,
       deckCards: null,
       deckList: null,
-      selectedDeck: null
+      selectedDeck: null,
+      searchFilter: null,
     };
   },
   components: {
     ResultsTable,
     DeckCardList,
-    DeckList
+    DeckList,
+    SearchFilterModal,
   },
   methods: {
+    SetSearchFilter(value) {
+      this.searchFilter = value;
```

Comment:
Either the button wording in the other component should be changed or this should trigger a search.